lib/core/include/irods/rcConnect.h

```diff
@@ -239,8 +231,10 @@ clientLoginPam( rcComm_t *conn, char *password, int ttl );
 int
 clientLoginTTL( rcComm_t *conn, int ttl );
 
-char *
-getSessionSignatureClientside();
+__attribute__((deprecated("Use session_signature member variable in RcComm instance.")))
```

Comment:
Should we mention `set_session_signature_client_side` here?

Reply:
It's not clear to me why we'd do that. Can you elaborate?

Reply:
Oh, that says "get" not "set". Nevermind.